velox/exec/Window.h

```diff
@@ -145,6 +145,9 @@ class Window : public Operator {
   // The decodedInputVectors_ are reused across addInput() calls to decode
   // the partition and sort keys for the above RowContainer.
   std::vector<DecodedVector> decodedInputVectors_;
+  // HashStringAllocator required by functions that allocate out of line
+  // string values.
```

Comment:
> string values.

Not necessarily "string" values. The name 'HashStringAllocator' is confusing as it reads as it allocates strings. It allocates byte buffers.